addons/hr_work_entry_holidays/tests/test_performance.py

```diff
@@ -111,7 +111,7 @@ def setUpClass(cls):
     def test_work_entries_generation_perf(self):
         # Test Case 7: Try to generate work entries for
         # a hundred employees over a month
-        with self.assertQueryCount(__system__=2607, admin=2807):
+        with self.assertQueryCount(__system__=500, admin=2807):
```

Comment:
:eyes:

Reply:
To be honest this has nothing to do with this branch :D I just updated several counters